src/helpers/assets.js

```diff
@@ -268,5 +319,47 @@ export const buildUniqueTokenList = (uniqueTokens, selectedShowcaseTokens) => {
   return rows;
 };
 
+const regex = RegExp(/\s*(the)\s/, 'i');
+
+export const buildBriefUniqueTokenList = (
```

Comment:
We need to be sure this function runs only when it's needed and all of the memorization where we use it works

Reply:
i am sure, i checked this